core/invoker/src/main/scala/whisk/core/containerpool/docker/DockerClient.scala

```diff
@@ -124,7 +124,7 @@ class DockerClient(dockerHost: Option[String] = None,
         }
         .map(ContainerId.apply)
         .recoverWith {
-          // https://docs.docker.com/v1.12/engine/reference/run/#/exit-status
+          // https://docs.docker.com/engine/reference/run/#general-form
```

Comment:
This is the proper link for exit status: https://docs.docker.com/engine/reference/run/#exit-status.